test/matrix/batch_dense_kernels.cpp

```diff
@@ -43,17 +44,28 @@ class Dense : public CommonTestFixture {
             std::normal_distribution<>(-1.0, 1.0), rand_engine, ref);
     }
 
-    void set_up_apply_data(gko::size_type num_rows, gko::size_type num_vecs = 1)
+    void set_up_apply_data(gko::size_type num_rows,
+                           gko::size_type num_cols = 32,
+                           gko::size_type num_vecs = 1)
```

Comment:
any usage for this changes?

Reply:
Its more general, but will be useful in https://github.com/ginkgo-project/ginkgo/pull/1528